src/Illuminate/Collections/Arr.php

```diff
@@ -951,4 +951,35 @@ public static function wrap($value)
 
         return is_array($value) ? $value : [$value];
     }
+
+    public static function toFilled(array $arr, $value, int $start = 0, int $end = null): array
+    {
+        $arrLength = count($arr);
```

Comment:
`Arr::random()` is the only method which stores `count($array)` into a local variable. It names this local variable `$count`.

I'd rename it to either `$count`, or `$length`. `$arrLength` feels misplaced, to me at least.

Reply:
I think the local variable makes sense. No reason to re-count the length every time, so it would be inefficient to not set it to a local var. 

As for the variable name, I'm a fan of described variables in general, so $arrLength makes sense if base var stays as $arr. if it is changed to $array, then use $arrayLength. 

Using just $length or $count I guess is fine because there is only one length variable, so nothing else to get it confused with.

Reply:
It's fine to me, it also makes sense to name it count.